python/src/aiconfig/editor/client/src/themes/LocalTheme.ts

```diff
@@ -74,6 +74,7 @@ export const LOCAL_THEME: MantineThemeOverride = {
       borderLeft: "none",
       borderTopRightRadius: "4px",
       borderBottomRightRadius: "4px",
+      minWidth: "32px",
```

Comment:
We should revisit later and move all the common styles to the component itself & just have overrides in the themes, otherwise if we add additional customization for theme props later we will force users of the component to need to duplicate a lot of the styles as well just to have the basic styles working (e.g. right now if there is not theme specified, the prompt cells are super narrow instead of expanding the full width because 'flex: 1' is defined in the theme styles instead of on the component